pages/_document.tsx

```diff
@@ -2,6 +2,10 @@ import type { PageProps } from '@prezly/theme-kit-nextjs';
 import { DEFAULT_LOCALE, getLocaleDirection, LocaleObject } from '@prezly/theme-kit-nextjs';
 import Document, { Head, Html, Main, NextScript } from 'next/document';
 
+/**
+ * Note that the attributes set here only apply on the initial page load.
+ * All subsequent locale changes are handled on the client side in `CulturePicker` component.
```

Comment:
There is no `CulturePicker` component 🤔

Reply:
Oh, thanks for the catch, forgot to rename it from the Privacy Portal :)